### PR TITLE
redirecting-pass

### PR DESCRIPTION
**Pull request by Javitocatral**

### CODE REVIEW

File: src/lib/assets/google-wallet-b.svg
Comment (zwergius) [inline src/lib/assets/google-wallet-b.svg:1]: svg's are responsive by definition, so with 1 icon it is sufficient

File: src/lib/assets/ios-wallet-big.svg
Comment (zwergius) [inline src/lib/assets/ios-wallet-big.svg:1]: 1 size is enough, resize with css

File: src/lib/components/link-wallet.svelte [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<script lang="ts">
+  export let href: string
+  export let src: string
+</script>
+
+<a {href}> <img {src} alt="" /></a>
```
Comment (zwergius) [inline src/lib/components/link-wallet.svelte:6]: this seems pointless

File: src/lib/components/link-wallet.svelte
Comment (zwergius) [inline src/lib/components/link-wallet.svelte:1]: this component only adds extra lines of code, delete

File: src/routes/[lang]/contact/christian/+page.svelte [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    if (isIOS || isMac) {
+      window.location.href = pkpass
+    } else if (isAndroid) {
+      window.location.href = passGoogle
+    }
+  }
+</script>
+
+<div class="add-wallet mobile">
+  <LinkWallet href={pkpass} src={downloadIos} />
```
Comment (zwergius) [inline src/routes/[lang]/contact/christian/+page.svelte:33]: change to anchor and probably use external

File: src/routes/[lang]/contact/christian/+page.svelte [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+  <LinkWallet href={passGoogle} src={downloadAndroid} />
+</div>
+<div class="add-wallet tablet">
+  <LinkWallet href={pkpass} src={downloadIosMediun} />
+  <LinkWallet href={passGoogle} src={downloadAndroidMediun} />
+</div>
+<div class="add-wallet desktoc">
```
Comment (zwergius) [inline src/routes/[lang]/contact/christian/+page.svelte:40]: typo

**Review by zwergius — CHANGES_REQUESTED** [review]

File: src/routes/[lang]/contact/christian/+page.svelte [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+  .add-wallet {
+    gap: 50px;
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+  .android {
+    width: 320px;
+    height: 89px;
```
Comment (zwergius) [inline src/routes/[lang]/contact/christian/+page.svelte:44]: magic numbers - why not just height: auto ? 

and then they can even share the class

**Review by zwergius — CHANGES_REQUESTED** [review]

File: src/routes/[lang]/contact/christian/+page.svelte (hunk 4 above)
Comment (Javitocatral) [reply src/routes/[lang]/contact/christian/+page.svelte:44]: well no, it is calculated with the reference size that the download svg gives you

Comment (zwergius) [reply src/routes/[lang]/contact/christian/+page.svelte:44]: auto is better

File: src/routes/[lang]/contact/christian/+page.svelte [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+<style>
+  .add-wallet {
+    gap: 50px;
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+  .android {
```
Comment (zwergius) [inline src/routes/[lang]/contact/christian/+page.svelte]: this class is duplicate of .iOS

File: src/routes/[lang]/contact/christian/+page.svelte [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+      window.location.href = pkpass
+    } else if (isAndroid) {
+      window.location.href = passGoogle
+    }
+  }
+</script>
+
+<div class="add-wallet">
+  <a href={pkpass}
+    ><img class="link-wallet" src={iosWallet} alt="button-ios" /></a
```
Comment (zwergius) [inline src/routes/[lang]/contact/christian/+page.svelte:29]: bad indentation
img tag is not a link either, so class name is wrong - it's best if it describes what it is so code makes sense... use "wallet-logo" for example
please study alt attribute and correct

**Review by zwergius — CHANGES_REQUESTED** [review]

File: src/routes/[lang]/contact/christian/+page.svelte [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+
+    if (isIOS || isMac) {
+      window.location.href = pkpass
+    } else if (isAndroid) {
+      window.location.href = passGoogle
+    }
+  }
+</script>
+
+<div class="add-wallet">
```
Comment (zwergius) [inline src/routes/[lang]/contact/christian/+page.svelte:27]: also bad class name

**Review by zwergius — CHANGES_REQUESTED** [review]

File: src/routes/[lang]/contact/christian/+page.svelte [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+  <a href={pkpass}
+    ><img class="link-wallet" src={iosWallet} alt="button-ios" /></a
+  >
+  <a href={passGoogle}
```
Comment (zwergius) [inline src/routes/[lang]/contact/christian/+page.svelte:31]: these links are external so needs to be defined, read sveltkit docs if you have doubts

**Review by zwergius — CHANGES_REQUESTED** [review]

File: src/routes/[lang]/contact/christian/+page.svelte [hunk 9]
```diff
@@ -24,38 +24,46 @@
...
-  <a href={pkpass}
-    ><img class="link-wallet" src={iosWallet} alt="button-ios" /></a
-  >
-  <a href={passGoogle}
-    ><img class="link-wallet" src={googleWallet} alt="button-android" /></a
-  >
+<div class="container">
+  <a rel="external" href={pkpass}>
+    <img
+      class="wallet-log"
```
Comment (zwergius) [inline src/routes/[lang]/contact/christian/+page.svelte:30]: typo

**Review by zwergius — CHANGES_REQUESTED** [review]

**Review by zwergius — APPROVED** [review]

**Pull request merged**